### PR TITLE
add actions support for fwfilter

### PR DESCRIPTION
**Pull request by chengzhycn**
Signed-off-by: chengzhycn <chengzhycn@gmail.com>

netlink library seems that don't support add actions for fwfilter now.

### CODE REVIEW

**Comment by chengzhycn** [comment]
@aboch could you help me review this pr?

**Comment by chengzhycn** [comment]
@vishvananda could you help me review this pr?

**Comment by chengzhycn** [comment]
@aboch Gentle ping. Thanks :)

File: filter_test.go [hunk 1]
```diff
@@ -510,6 +510,127 @@ func TestFilterFwAddDel(t *testing.T) {
...
+	mia, ok := fw.Actions[0].(*MirredAction)
+	if !ok {
+		t.Fatal("Unable to find mirred action")
+	}
+
+	if mia.Attrs().Action != TC_ACT_STOLEN {
+		t.Fatal("Mirred action isn't TC_ACT_STOLEN")
+	}
+
+	if err := FilterDel(filter); err != nil {
```
Comment (aboch) [inline filter_test.go:611]: Can you please also test the `.MirredAction == TCA_EGRESS_REDIR` ?

**Comment by aboch** [comment]
LGTM

**Pull request merged**